po/es.po

```diff
@@ -88,7 +88,7 @@ msgstr "Firmware"
 
 #: lib/logitech_receiver/i18n.py:48
 msgid "Hardware"
-msgstr "Hardware"
+msgstr "Material"
```

Comment:
o_O - I am not sure what happened there, I fully agree about "Hardware"! Feel free to revert that change, and thanks for your other comment.